docs/javascript/v2/how-to-guides/set-up-video-conferencing/captions.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Captions (Transcription) (Beta)
+nav: 1.24
+---
+
+You can implement captions (subtitles) in your WEB conferencing app easily with 100ms SDK. The SDK provides a callback with transcript for each peer when they speak.
```

Comment:
100ms' real-time transcription engine generates a live transcript (closed captions) during a conferencing session. The SDK provides a callback with transcript for each peer when they speak.